tasks/gulp/__tests__/check-compatibility-with-govuk-frontend-version.test.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-env jest */
+const { writeFileSync, unlink } = require('fs');
+const path = require('path');
+const { exec } = require('child_process');
+
+const initCwd = path.join(process.env.INIT_CWD, 'tasks/gulp/__tests__');
+
+describe('govuk-frontend version compatibility check', () => {
+  const scriptPath = path.resolve(__dirname, '../../../check-compatibility-with-govuk-frontend-version.js');
+  const mockPackagePath = path.resolve(__dirname, 'package.json');
+  const createMockPackage = (contents) => writeFileSync(mockPackagePath, JSON.stringify(contents));
+
+  afterAll(() => {
+    unlink(mockPackagePath, () => {});
+  });
+
+  beforeAll(() => {
+    process.env.HMRC_FRONTEND_DISABLE_COMPATIBILITY_CHECK = '';
+  });
+
+  describe('Installing outside of a Prototype kit project', () => {
+    it('should exit the process with code 0', (done) => {
+      createMockPackage({
+        dependencies: {
+          'govuk-frontend': '5.11.0',
```

Comment:
I think here we should use an incompatible version for the test to show that "even though using incompatible version, because you're outside prototype kit, it won't do anything"

```suggestion
          'govuk-frontend': '4.4.0',
```